addons/mail/static/src/core/common/autoresize_input.js

```diff
@@ -38,7 +40,9 @@ export class AutoresizeInput extends Component {
             if (downTarget === this.inputRef.el) {
                 return;
             }
-            this.onValidate();
+            if (this.focused) {
```

Comment:
external click seems like a bad way, why not using blur or similar event instead?

Or directly validating in focus out?